lua/catppuccin/groups/integrations/treesitter_context.lua

```diff
@@ -1,7 +1,18 @@
 local M = {}
 
 function M.get()
-	return O.transparent_background and { TreesitterContextBottom = { sp = C.dim, style = { "underline" } } } or {}
+	return O.transparent_background and {
+		TreesitterContextBottom = { sp = C.dim, style = { "underline" } },
+	} or {
+		TreesitterContextBottom = {
+			sp = C.surface0,
```

Comment:
Shouldn't this be fg instead of sp?

# With sp

![image](https://github.com/catppuccin/nvim/assets/56817415/67aac51b-2e34-447f-86b7-e835ab081f67)

# With fg

![image](https://github.com/catppuccin/nvim/assets/56817415/d29a2675-b0b2-47b0-ab27-b1a444a6dd06)

Reply:
You may be using WezTerm. It seems that WezTerm does not support the underline of having another color for the word. In the terminal I used, Kitty(what I'm using), Alacritty, Gnome Terminal, and Konsole all support this feature but WezTerm does not.

Reply:
![image](https://github.com/catppuccin/nvim/assets/79978224/d125898e-4cd1-4e8b-a88a-faf79709746a)
I'm using the latest nightly build of WezTerm, I think this looks as it should? @ofseed

Reply:
I'm sorry I misread the PR, I thought that was *for* transparency.
Here's the PR with transparency disabled, in WezTerm:
![image](https://github.com/catppuccin/nvim/assets/79978224/c9f5459e-f263-41f7-9970-c5366c83adc6)

Reply:
Quoting from [wezterm faq](https://wezfurlong.org/wezterm/faq.html#how-do-i-enable-undercurl-curly-underlines):

Note: on Windows, the ConPTY layer strips out the curly underline escape sequences. If you're missing this feature in your WSL instance, you will need to use either `wezterm ssh` or [multiplexing](https://wezfurlong.org/wezterm/multiplexing.html#connecting-into-windows-subsystem-for-linux) to bypass ConPTY.

Reply:
I think we can mark this as resolved, then?